packages/vuetify/src/components/VLazy/VLazy.tsx

```diff
@@ -0,0 +1,70 @@
+// Composables
+import { makeDimensionProps, useDimension } from '@/composables/dimensions'
+import { makeProps, maybeTransition } from '@/util'
+import { makeTagProps } from '@/composables/tag'
+import { makeTransitionProps } from '@/composables/transition'
+import { useProxiedModel } from '@/composables/proxiedModel'
+
+// Directives
+import intersect from '@/directives/intersect'
+
+// Utilities
+import { defineComponent, PropType } from 'vue'
+
+export default defineComponent({
+  name: 'VLazy',
+
+  directives: { intersect },
+
+  props: makeProps({
+    modelValue: Boolean,
+    options: {
+      type: Object as PropType<IntersectionObserverInit>,
+      // For more information on types, navigate to:
+      // https://developer.mozilla.org/en-US/docs/Web/API/Intersection_Observer_API
+      default: () => ({
+        root: undefined,
+        rootMargin: undefined,
+        threshold: undefined,
+      }),
+    },
+    ...makeDimensionProps(),
+    ...makeTagProps(),
+    ...makeTransitionProps({ transition: 'fade-transition' }),
+  }),
+
+  emits: {
+    'update:modelValue': (value: boolean) => true,
+  },
+
+  setup (props, { slots }) {
+    const { dimensionStyles } = useDimension(props)
+    const isActive = useProxiedModel(props, 'modelValue')
+
+    function onIntersect (isIntersecting: boolean) {
+      if (isActive.value) return
+
+      isActive.value = isIntersecting
+    }
+
+    return () => (
+      <props.tag
+        class="v-lazy"
+        v-intersect={[
+          onIntersect,
+          props.options,
+          ['once'],
```

Comment:
if it really is intended to work only once then `modelValue` doesn't make much sense imho

```vue
<template>
  <v-app>
    <v-btn style="position: fixed" @click="model = false">Reset (model={{ model }})</v-btn>
    <div class="ma-12 px-12" style="padding-top: 150vh !important; padding-bottom: 100vh !important;">
      <v-lazy v-model="model" height="200">
        <v-sheet elevation="2">
          <div class="pa-4">
            Hello text
          </div>
        </v-sheet>
      </v-lazy>
    </div>
  </v-app>
</template>

<script>
  export default {
    data: () => ({ model: false }),
  }
</script>
```